components/WordsForm.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import Image from "next/image";
+import iconCheckmark from "../public/assets/voc-trainer_icon_checkmark.png";
+import iconEye from "../public/assets/voc-trainer_icon_eye.png";
+import { BiEdit, BiSave, BiTrash, BiXCircle } from "react-icons/bi";
```

Comment:
`BiSave` and `BiXCircle` are for future use?

Reply:
yes